Convert fbv package to DOWNLOAD helper.

Not needed on recent BR.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/package/fbv/fbv.mk
===================================================================
--- buildroot.orig/package/fbv/fbv.mk	2011-03-01 13:57:26.000000000 +0100
+++ buildroot/package/fbv/fbv.mk	2011-03-01 13:59:27.000000000 +0100
@@ -12,7 +12,7 @@
 FBV_TARGET_BINARY:=usr/bin/$(FBV_BINARY)
 
 $(DL_DIR)/$(FBV_SOURCE):
-	$(WGET) -P $(DL_DIR) $(FBV_SITE)/$(FBV_SOURCE)
+	$(call DOWNLOAD,$(FBV_SITE),$(FBV_SOURCE))
 
 fbv-source: $(DL_DIR)/$(FBV_SOURCE)
 
